minter-suite/custom-minters.md

```diff
@@ -0,0 +1,61 @@
+---
+order: 850
+---
+
+# Custom, One-Off Minters
+
+Some Engine partners may wish to add custom, one-off minters to the shared minter suite. This is able to be done by the admin of an Engine partner's core contract.
+
+Note that due to the high flexibility available to custom minters, they are not indexed by the Art Blocks subgraph, and therefore are not able to be configured by artists via the Art Blocks frontend. Instead, the custom minter will need to be configured via the Engine partner's frontend, etherscan, etc.
```

Comment:
Should highlight this in a callout imo

Reply:
good call, updated in [f1deab9](https://github.com/ArtBlocks/artblocks-docs/pull/299/commits/f1deab981440c2939480a56f24c1d67a1a0540b4)